src/Annotation/ApiResource.php

```diff
@@ -129,6 +130,13 @@
      */
     private $forceEager;
 
+    /**
+     * @see https://github.com/Haehnchen/idea-php-annotation-plugin/issues/112
```

Comment:
In my PR (https://github.com/api-platform/core/pull/1685) it was mentioned that the core team does not want to add more root annotation configuration values but instead put it into `attributes`. Just saying that this may need to be modified but I let @dunglas decide on this. I don't remember the exact reason for this anymore because I actually like when things are specific 😄

Reply:
this just follows https://github.com/api-platform/core/pull/1788, nothing more